tailwind.config.js

```diff
@@ -7,6 +7,15 @@ module.exports = {
         'max-height': 'max-height',
         'backdrop-filter': 'backdrop-filter',
       },
+      keyframes: {
+        slide: {
+          '0%': { transform: 'translateX(0)' },
+          '100%': { transform: 'translateX(-100%)' },
+        },
+      },
+      animation: {
+        slide: '20s slide infinite linear',
+      },
```

Comment:
Have we check if we have it in the `quill-tokens`? Maybe we can use it directly in `classNames`

Reply:
checked with @mohsen-deriv this should be fine for now

Reply:
we don't have the motions in tailwind classes yet, we're gonna work on it soon.